var/spack/repos/builtin/packages/py-contextvars/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyContextvars(PythonPackage):
+    """This package implements a backport of Python 3.7 contextvars module (see
+    PEP 567) for Python 3.6."""
+
+    homepage = "http://github.com/MagicStack/contextvars"
+    pypi = "contextvars/contextvars-2.4.tar.gz"
+
+    version("2.4", sha256="f38c908aaa59c14335eeea12abea5f443646216c4e29380d7bf34d2018e2c39e")
+
+    depends_on("python@3.6:", type=("build", "run"))
```

Comment:
Where do you see this? I would just remove this line, Spack only supports 3.7+ anyway